tests/integration/features/example.feature

```diff
@@ -0,0 +1,33 @@
+@example
```

Comment:
Why name as example? It is temporary?

Reply:
@budhrg temporary, will then probably go for `basic` or something like that.